pkg/controller/mizar/mizar-pod-controller.go

```diff
@@ -103,7 +103,7 @@ func (c *MizarPodController) Run(workers int, stopCh <-chan struct{}) {
 
 func (c *MizarPodController) createObj(obj interface{}) {
 	key, _ := controller.KeyFunc(obj)
-	c.queue.Add(KeyWithEventType{Key: key, EventType: EventType_Create})
+	c.queue.Add(KeyWithEventType{Key: key, EventType: EventType_Create, ResourceVersion: getResourceVersion(obj)})
```

Comment:
Do we need a resourceVersion here as there is only one create event for each pod?

If that is the case, we can just put an empty string as the ResourceVersion.

Reply:
Updated